charts/routes/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+CaraML Routes Notes:
+.....
```

Comment:
should we print the console UI link here?

Reply:
i forgot about that, have added the notes, sample output:
```
➜ helm upgrade routes charts/routes --namespace mlp
Release "routes" has been upgraded. Happy Helming!
NAME: routes
LAST DEPLOYED: Tue Oct 11 17:25:19 2022
NAMESPACE: mlp
STATUS: deployed
REVISION: 10
TEST SUITE: None
NOTES:
CaraML Routes Notes:

* Console UI can be accessed at [console.35.201.218.244.nip.io]


* Docs can be accessed at:
- docs.35.201.218.244.nip.io/turing/rest-api
- docs.35.201.218.244.nip.io/merlin/rest-api
- docs.35.201.218.244.nip.io/xp/rest-api
- docs.35.201.218.244.nip.io/xp/treatment-api
```